spec/views/foods/create.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe 'foods/create.html.erb', type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
[Optional] Since create page isn't used by the app, you can delete the integration test for it too